Jint/Runtime/TypeConverter.cs

```diff
@@ -490,6 +490,8 @@ private static string ToStringNonString(JsValue o)
                     return Null.Text;
                 case InternalTypes.Object when o is IPrimitiveInstance p:
                     return ToString(ToPrimitive(p.PrimitiveValue, Types.String));
+                case InternalTypes.Object when o is Interop.IObjectWrapper p:
+                    return ToString(ToPrimitive(p.Target?.ToString(), Types.String));
```

Comment:
I'm not sure if the ToPrimitive is needed here as we know that ToString produces string? Basically could just be p.Target?.ToString()?

Reply:
Thanks, didn't if it would do something special, changing